app/Services/PilotService.php

```diff
@@ -55,4 +55,9 @@ function () {
         return $pilot;
     }
 
+    public function create()
```

Comment:
Yeah move the actual registration code, after the `validate()` and above the view stuff into here. Then we can send the validation email, etc from here as well.